.bazelignore

```diff
@@ -1 +1,3 @@
 node_modules
+tools/dashboard/node_modules
+tools/dashboard/functions/node_modules
```

Comment:
Dashboard? 🤔

Reply:
Yeah I don't know where this came from, there's nothing under `tools/dashboard` other than `node_modules` directories.

I was seeing some issues when running `bazel build ...` that seemed to be fixed by the highlight.js stuff I added